src/Tizen.NUI/src/public/DragAndDrop/DragAndDrop.cs

```diff
@@ -54,12 +60,42 @@ private DragAndDrop(global::System.IntPtr cPtr, bool cMemoryOwn) : base(cPtr, cM
         /// <param name="sourceView">The soruce view</param>
         /// <param name="shadowView">The shadow view for drag object</param>
         /// <param name="dragData">The data to send</param>
+        /// <param name="callback">The source event callback</param>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public void StartDragAndDrop(View sourceView, View shadowView, DragData dragData)
+        public void StartDragAndDrop(View sourceView, View shadowView, DragData dragData, SourceEventHandler callback)
         {
-            if (!Interop.DragAndDrop.StartDragAndDrop(SwigCPtr, View.getCPtr(sourceView), View.getCPtr(shadowView), dragData.MimeType, dragData.Data))
+            if (mDragWindow)
             {
-                throw new InvalidOperationException("Fail to StartDragAndDrop");
+                mDragWindow.Hide();
+                mDragWindow.Dispose();
+                mDragWindow = null;
+            }
+
+            //FIXME: Drag window cannot be reused because the current window server does not allow reuse.
+            //       When window server allows the drag window to be reused, makes the mDragWindow created once.
+            mDragWindow = new Window("DragWindow", new Rectangle(-shadowWidth, -shadowHeight, shadowWidth, shadowHeight), true)
```

Comment:
Could you remove this warning message?
If not, please add 'suppress tag' for the message.